### PR TITLE
Add support for names such as: name[] and name[abc]

### PR DESCRIPTION
**Pull request by IsmailM**
Further to #18 

Tested the following: https://codepen.io/ismailm___/pen/QzEbyZ?editors=1010

### CODE REVIEW

File: conditionize.flexible.jquery.js [hunk 1]
```diff
@@ -44,7 +44,7 @@
       // Replace fields names/ids by $().val()
       for (name in inputs) {
         e = inputs[name];
-        tmp_re = new RegExp("(" + name + ")\\b" + ifNotInQuotes, 'g')
+        tmp_re = new RegExp("(" + name.replace('\[', '\\[').replace('\]', '\\]') + ")\\s" + ifNotInQuotes, 'g')
```
Comment (IsmailM) [inline conditionize.flexible.jquery.js]: as you can see here, I have changed the word boundary (`\b`) to a space boundary (`\s`) character...

So the following will not work (which I think did before):

```
inputname=='foo' 

# This now needs to be rewritten as follows
inputname == 'foo' 

(so probably need to clarify that in the Readme)
```

Comment (CodeCurosity) [reply conditionize.flexible.jquery.js]: Yes it worked before, now i need to find a way to add space in arraymap and array combine, more work for me , lol

**Comment by rguliev** [comment]
It does not work. See [my comment](https://github.com/renvrant/conditionize.js/issues/18#issuecomment-447653622)

**Comment by IsmailM** [comment]
The same issue is also present in the current master branch: [https://codepen.io/ismailm___/pen/QzEjgj?editors=1010](https://codepen.io/ismailm___/pen/QzEjgj?editors=1010)

The issue is that checkboxes 

I guess the fix would probably be checking if it's a checkbox before running `fn.showOrHide()` and then if it's a checkbox, extract all checked values and then do the match...

**Comment by IsmailM** [comment]
Not sure what you mean by the second point in your [comment](https://github.com/renvrant/conditionize.js/issues/18#issuecomment-447653622)?

**Pull request closed without merging**